static/src/stylesheets/module/_adslot.scss

```diff
@@ -120,14 +120,14 @@
 }
 .top-banner-ad-container--above-nav {
     @include mq(tablet) {
-        .js-on & {
+        .js-on:not(.is-adfree) & {
```

Comment:
It's from the preflight scripts. See https://github.com/guardian/frontend/pull/10909